### PR TITLE
Feat/411 5 3 hide get block info get blockchain info

### PR DESCRIPTION
**Pull request by vorg-code**
Depends on:
---
#474 

Description
---
`getBlockInfo` and `getBlockchainInfo` are no longer available in blockchainService file to be used by the rest of the application, they are too blockchain specific. `getLastBlockTimestamp` was created in their place since it is the only use of those methods to get the time of the last block.

Test plan
---
These functions are used only in the network page, so, to test that everything is still working as usual:
1. Change ’grpc’ to ‘chronik’ for XEC in constants/index.ts, like this:
export const NETWORK_BLOCKCHAIN_CLIENTS: KeyValueT<BLOCKCHAIN_CLIENT_OPTIONS> = {
ecash: `'chronik'`,
bitcoincash: 'grpc'
}
2. open: http://localhost:3000/networks

### CODE REVIEW

**Review by Klakurka — DISMISSED** [review]

**Comment by Klakurka** [comment]
Conflicts.

**Comment by vorg-code** [comment]
@Klakurka fixed

**Review by Klakurka — DISMISSED** [review]

File: services/blockchainService.ts [hunk 1]
```diff
@@ -80,12 +80,11 @@ export async function syncTransactionsForAddress (parameters: GetAddressTransact
...
 
-export async function getBlockchainInfo (networkSlug: string): Promise<BlockchainInfo> {
-  return await getObjectValueForNetworkSlug(networkSlug, BLOCKCHAIN_CLIENTS).getBlockchainInfo(networkSlug)
-}
-
-export async function getBlockInfo (networkSlug: string, height: number): Promise<BlockInfo> {
-  return await getObjectValueForNetworkSlug(networkSlug, BLOCKCHAIN_CLIENTS).getBlockInfo(networkSlug, height)
+export async function getLastBlockTimestamp (networkSlug: string): Promise<number> {
+  const client = getObjectValueForNetworkSlug(networkSlug, BLOCKCHAIN_CLIENTS)
+  const getBlockchainInfo = await client.getBlockchainInfo(networkSlug)
```
Comment (chedieck) [inline services/blockchainService.ts:85]: I think this variable could be called `blockchainInfo`, without the `get`, for better clarity

**Review by chedieck — CHANGES_REQUESTED** [review]
LGTM, just a minor thing

File: services/blockchainService.ts (hunk 1 above)
Comment (vorg-code) [reply services/blockchainService.ts]: indeed, name improved

**Review by chedieck — APPROVED** [review]

**Pull request merged**